home-application/src/main/java/com/softserveinc/ita/homeproject/application/provider/JerseyParameterNameProvider.java

```diff
@@ -0,0 +1,68 @@
+package com.softserveinc.ita.homeproject.application.provider;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Method;
+import java.lang.reflect.Parameter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import javax.ws.rs.CookieParam;
+import javax.ws.rs.FormParam;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.MatrixParam;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Context;
+
+import org.hibernate.validator.internal.engine.DefaultParameterNameProvider;
+
+// Copy from https://github.com/zxqhoho/dropwizard/blob/26930f772f3e821d33382969d773825b326d49d5/dropwizard-jersey/src/main/java/io/dropwizard/jersey/validation/JerseyParameterNameProvider.java
```

Comment:
do we need this information?

Reply:
You are absolutely right. But perhaps we will need it in the future. I discussed this with a technical expert.

Reply:
Looks strange) 
if this we can't write on some file, ok no problem

Reply:
Just make it more reader friendly =) We have to provide information that this file isn't a part of our source code but was taken from another library that we cannot import though. And leave a recommendation to not change this file manually but update it from the source if any changed there.